Binance.Net/SubClients/BinanceClientSubAccounts.cs

```diff
@@ -1047,6 +1046,61 @@ public async Task<WebCallResult<BinanceSubAccountBlvt>> EnableBlvtForSubAccountA
             return await _baseClient.SendRequestInternal<BinanceSubAccountBlvt>(_baseClient.GetUrlSpot(subAccountEnableBlvtEndpoint, "sapi", "1"), HttpMethod.Post, ct, parameters, true).ConfigureAwait(false);
         }
 
+        #endregion
+
+        #region Query Universal Transfer History (For Master Account)
+
+        /// <summary>
+        /// Gets a list of universal transfers
+        /// </summary>
+        /// <param name="fromEmail">Filter the list by from email (fromEmail and toEmail cannot be present at same time)</param>
+        /// <param name="toEmail">Filter the list by to email (fromEmail and toEmail cannot be present at same time)</param>
+        /// <param name="startTime">Filter by start time</param>
+        /// <param name="endTime">Filter by end time</param>
+        /// <param name="page">The page of the results</param>
+        /// <param name="limit">The max amount of results to return (Default 500, max 500)</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <param name="ct">Cancellation token</param>
+        /// <returns>List of universal transfers</returns>
+        public WebCallResult<IEnumerable<BinanceSubAccountUniversalTransferTransaction>> GetUniversalTransferHistory(string? fromEmail = null, string? toEmail = null, long? startTime = null, long? endTime = null, int? page = null, int? limit = null, int? receiveWindow = null, CancellationToken ct = default) => GetUniversalTransferHistoryAsync(fromEmail, toEmail, startTime, endTime, page, limit, receiveWindow, ct).Result;
+
+        /// <summary>
+        /// Gets a list of universal transfers
+        /// </summary>
+        /// <param name="fromEmail">Filter the list by from email (fromEmail and toEmail cannot be present at same time)</param>
+        /// <param name="toEmail">Filter the list by to email (fromEmail and toEmail cannot be present at same time)</param>
+        /// <param name="startTime">Filter by start time</param>
+        /// <param name="endTime">Filter by end time</param>
+        /// <param name="page">The page of the results</param>
+        /// <param name="limit">The max amount of results to return (Default 500, max 500)</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <param name="ct">Cancellation token</param>
+        /// <returns>List of universal transfers</returns>
+        public async Task<WebCallResult<IEnumerable<BinanceSubAccountUniversalTransferTransaction>>> GetUniversalTransferHistoryAsync(string? fromEmail = null, string? toEmail = null, long? startTime = null, long? endTime = null, int? page = null, int? limit = null, int? receiveWindow = null, CancellationToken ct = default)
```

Comment:
startTime and endTime should be of type DateTime? to make it easier when calling. In the parameters you'll then have to convert from DateTime to long using a converter. Have a look at how it is done in other calls.